harness/scoring/judge_anchored.py

```
@@ -129,7 +207,13 @@ def _parse_retry_after(msg: str) -> float:
             try:
                 # OpenAI client timeout is set in _client(), but add explicit timeout as backup
                 api_timeout = float(os.getenv("OPENAI_JUDGE_TIMEOUT", os.getenv("OPENAI_TIMEOUT", "60.0")))
-                resp = client.chat.completions.create(**params, timeout=api_timeout)
+                if use_responses_api:
+                    resp = client.responses.create(timeout=api_timeout, **params)
+                else:
+                    resp = client.chat.completions.create(**params, timeout=api_timeout)
+                    # Validate response before breaking
+                    if resp is None or not hasattr(resp, "choices") or not resp.choices:
+                        raise ValueError(f"Invalid response from API: {type(resp).__name__}")
```

Ruff (0.14.3)

Abstract `raise` to an inner function

(TRY301)